torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -121,6 +129,14 @@ SizeDiv::SizeDiv(torch::lazy::Value a, torch::lazy::Value b)
   upper_bound_ = dim_node_0->getStaticValue() / dim_node_1->getStaticValue();
 };
 
+int64_t SizeDiv::getDynamicValue() const {
+  const torch::lazy::DimensionNode* dim_node_0 = DimCast(operand(0));
+  const torch::lazy::DimensionNode* dim_node_1 = DimCast(operand(1));
+  XLA_CHECK(dim_node_0);
+  XLA_CHECK(dim_node_1);
+  return dim_node_0->getDynamicValue() / dim_node_1->getDynamicValue();
```

Comment:
Why do we not have the following check?

```
XLA_CHECK(dim_node_1->getDynamicValue() != 0)
      << "Can't divide a dimension by zero";
```

CC @vanbasten23 @JackCaoG

Reply:
yea, I guess we can have this check

Reply:
@vanbasten23 can you please add the PR?

Reply:
Thanks for pointing it out. Working on it.